### PR TITLE
[single-staking-pools-balanceof] New Strategy for Single Staking Pools

### PR DESCRIPTION
**Pull request by PolySwift**
Implements new strategy to fetch Staked token amount in single staking pool

Changes proposed in this pull request:
- Implementes new single-staking-pools-balanceof

### CODE REVIEW

File: src/strategies/single-staking-pools-balanceof/index.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable prettier/prettier */
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'bonustrack';
```
Comment (ChaituVR) [inline src/strategies/single-staking-pools-balanceof/index.ts:5]:
```suggestion
export const author = 'PolySwift';
```

File: src/strategies/single-staking-pools-balanceof/index.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+export const version = '0.1.1';
+
+// const abi = [
+//   'function balanceOf(address account) external view returns (uint256)'
+// ];
```
Comment (ChaituVR) [inline src/strategies/single-staking-pools-balanceof/index.ts:10]:
```suggestion
```

File: src/strategies/single-staking-pools-balanceof/index.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable prettier/prettier */
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'bonustrack';
+export const version = '0.1.1';
```
Comment (ChaituVR) [inline src/strategies/single-staking-pools-balanceof/index.ts:6]:
```suggestion
export const version = '0.1.0';
```

File: src/strategies/single-staking-pools-balanceof/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# single-staking-pools-balanceof
+
+Used for fetching the staked token balance in a single staking pool
+
+Here is an example of parameters:
+
+```json
+{
+  "stakingPoolAddress": "0x6b175474e89094c44da98b954eedeac495271d0f",
```
Comment (ChaituVR) [inline src/strategies/single-staking-pools-balanceof/README.md:9]:
```suggestion
  "stakingPoolAddress": [
      "0x081Ffa6Fa76e738531B3717301F4B636efAe1F1e"
   ],
```

File: src/strategies/single-staking-pools-balanceof/examples.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "single-staking-pools-balanceof",
+      "params": {
+        "stakingPoolAddresses": [
+          "0x081Ffa6Fa76e738531B3717301F4B636efAe1F1e"
+        ],
+        "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
+        "symbol": "DAI",
```
Comment (ChaituVR) [inline src/strategies/single-staking-pools-balanceof/examples.json:11]:
```suggestion
        ],
        "symbol": "DAI",
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by PolySwift** [comment]
@ChaituVR I see you've helped me to update the changes you requested, is there anything else you need from my side?

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
@PolySwift 
Strategy is live: https://snapshot.org/#/strategy/single-staking-pools-balanceof

**Pull request merged**